modules/administration-guide/partials/con_external-postgresql-setup.adoc

```diff
@@ -0,0 +1,39 @@
+[id="external-postgresql-setup_{context}"]
+= External PostgreSQL setup
+
+The {prod-short} server uses a PostgreSQL database for persisting data about the state of {prod-short}. The database contains information about user accounts, workspaces, preferences, and other details.
+
+By default, the {prod-short} Operator creates and manages deployment of this database. However, the {prod-short} Operator does not support full life-cycle capabilities such as backups and recovery.
+
+The following disaster-recovery options are recommended when configuring an external PostgreSQL database for a business-critical setup:
+
+* High Availability (HA)
+* Point In Time Recovery (PITR)
+
+The external PostgreSQL instance can be configured as on-premises or to use a cloud service such as Amazon Relational Database Service (Amazon RDS). Amazon RDS enables deployment of production databases in a Multi-Availability Zone configuration for a resilient disaster recovery strategy with daily and on-demand snapshots.
+
+The recommended configuration of an example database:
+
+[options="header"]
+|===
+|Parameter | Value
+|Instance class | db.t2.small
+|vCPU | 1
+|RAM | 2 GB
+|Multi-az | true, 2 replicas
+|Engine version | 9.6.11
+|TLS | enabled
+|Automated backups | enabled (30 days)
+|===
+
+Making workspace metadata and user information persistent involves configuring both the external PostgreSQL and Che:
```

Comment:
PostgreSQL database is used to persist users' data. If the database stored in the cluster, it might be deleted when deleting Che. In case of external database server, this is independent process.